.github/workflows/main.yml

```diff
@@ -400,7 +400,34 @@ jobs:
         run: mvn -B package -DskipTests=true
       - name: StoneDB Tests
         run: |
-          STONEDB_AVAILABLE=true mvn test -Dtest=TestStoneDB
+          STONEDB_AVAILABLE=true mvn test -Dtest=TestStoneDBNoRec
+
+  stonedb-tlp:
+    name: DBMS Tests (StoneDB TLP)
```

Comment:
@mrigger Is it necessary to include two oracles for a database in CI? I saw other databases only have one.

Reply:
I think either is fine.